error.handling.js

```diff
@@ -12,13 +12,10 @@ exports.custumError = (err, req, res, next) => {
   next(err);
 };
 
-// exports.unmatchRouts = (err, req, res, next) => {
-//   console.log("I am here in unmatchRouts");
-//   res.status(404).send({ msg: "Not found" });
-
-//   //   next(err);
-// };
-
+exports.unmatchRouts = (req, res, next) => {
+  console.log("Unmatched route triggered:", req.originalUrl);
+  res.status(404).send({ msg: "Not found" });
+};
 exports.serverError = (err, req, res, next) => {
   res.status(500).send({ msg: "Server Error" });
```

Comment:
Really nice including a server error response! 🥇  But with TDD in mind, how would we test for this?